### ISSUE TITLE
Driver doesn't update since dropping official insatller in v8824

### ISSUE THREAD

**Issue by jlpetz** [pal1000/Realtek-UAD-generic#23]
I beleive the last time I installed the pal1000 driver was version 6.0.8824.1 which came with the offical installer. 

Since then using the unofficial driver setup/install the driver does not seem to get installed as I'm stilll on 8824. I just tried 6.0.8849.1 rebooted and same thing. I didn't notice any error ouput during the bat script setup

Any tips on getting you debug info on why it's not pushing the updated driver to the system

Windows 10 64bit build 1909

**Comment by jlpetz** [comment]
```
Begin uninstalling Realtek UAD driver...

Stopping Windows Audio service to reduce reboot likelihood...

Done.

Removing Realtek Audio Universal Service registration record...

Done.

Terminating per-user instances of Realtek Audio Universal Service...

Done.

HDAUDIO\FUNC_01&VEN_10EC&DEV_1220&SUBSYS_1458A0CD&REV_1001\5&26C86C12&0&0001: Disabled
1 device(s) disabled.

No matching devices found.

HDAUDIO\FUNC_01&VEN_10EC&DEV_1220&SUBSYS_1458A0CD&REV_1001\5&26C86C12&0&0001: Removed
1 device(s) were removed.

No devices were removed.

Microsoft PnP Utility

Driver package deleted successfully.

Microsoft PnP Utility

Driver package deleted successfully.

Microsoft PnP Utility

Driver package deleted successfully.

Microsoft PnP Utility

Driver package deleted successfully.

Microsoft PnP Utility

Driver package deleted successfully.

Microsoft PnP Utility

Driver package deleted successfully.

The Windows Audio service is starting.
The Windows Audio service was started successfully.


Done uninstalling driver.

Do you want to install unofficial and minimal Realtek UAD generic package (y/n):y

Microsoft PnP Utility

Adding driver package:  Win64\HDXRT.inf
Driver package added successfully.
Published Name:         oem2.inf

Adding driver package:  Win64\HDXRTSST.inf
Driver package added successfully.
Published Name:         oem9.inf

Adding driver package:  Win64\Realtek\ExtRtk_8824\HDX_GenericExt_RTK.inf
Driver package added successfully.
Published Name:         oem21.inf

Adding driver package:  Win64\Realtek\RealtekAPO_736\RealtekAPO.inf
Driver package added successfully.
Published Name:         oem27.inf

Adding driver package:  Win64\Realtek\RealtekHSA_204\RealtekHSA.inf
Driver package added successfully.
Published Name:         oem28.inf

Adding driver package:  Win64\Realtek\RealtekService_220\RealtekService.inf
Driver package added successfully.
Published Name:         oem29.inf

Total driver packages:  6
Added driver packages:  6

Done installing driver

Press any key to continue . . .

Enabling Windows advanced startup recovery menu in case something goes very wrong...
The operation completed successfully.

Scanning for new hardware.
Scanning completed.

Give Windows 20 seconds to load Realtek UAD driver...
Press any key to continue . . .

Reverting Windows to normal startup...
The operation completed successfully.

Press any key to continue . . .

-----------------------------------------------------
This tool will attempt to forcefully update Realtek UAD generic driver codec core component by replacing
older driver files with newer version. It is intended to run only after performing a driver update with setup.cmd.

Press any key to continue . . .
-----------------------------------------------------
Begin force update procedure...

HDAUDIO\FUNC_01&VEN_10EC&DEV_1220&SUBSYS_1458A0CD&REV_1001\5&26C86C12&0&0001: Disabled
1 device(s) disabled.

No matching devices found.

Win64\Realtek\UpdatedCodec\RTAIODAT.DAT
Win64\Realtek\UpdatedCodec\RTKVHD64.sys
        2 file(s) copied.

Enabling Windows advanced startup recovery menu in case something goes very wrong...
The operation completed successfully.


HDAUDIO\FUNC_01&VEN_10EC&DEV_1220&SUBSYS_1458A0CD&REV_1001\5&26C86C12&0&0001: Enabled
1 device(s) are enabled.

No matching devices found.

Give Windows 10 seconds to load Realtek UAD driver...
Press any key to continue . . .

Reverting Windows to normal startup...
The operation completed successfully.

Press any key to continue . . .
-----------------------------------------------------
```

**Comment by jlpetz** [comment]
Hmmm, maybe I take that back. It's kinda installed I guess? The driver info in device manager still says 6.0.8824.1 but if you click into the 'Driver Details' button it shows RTKVHD64.sys is actually 6.0.8849.1 but is missing the pretty certificate icon, so maybe it's reverting to my last good driver?

![Realtek8849](https://user-images.githubusercontent.com/16551502/69723033-a4f6cf80-116c-11ea-8acb-c37167b4d73c.PNG)

**Comment by jlpetz** [comment]
Should the driver package have two copies of 'RTKVHD64.sys' both with what appears to be valid digital signatures

I see one in 
\Unofficial-Realtek-UAD-generic-6.0.8849.1\Realtek-UAD-generic\Win64\RTKVHD64.sys
![Realtek_1](https://user-images.githubusercontent.com/16551502/69723573-079c9b00-116e-11ea-9c54-d7b71614b3d4.PNG)

and another in
\Unofficial-Realtek-UAD-generic-6.0.8849.1\Realtek-UAD-generic\Win64\Realtek\UpdatedCodec
![Realtek_2](https://user-images.githubusercontent.com/16551502/69723583-0bc8b880-116e-11ea-8473-93560ec30c8e.PNG)

**Comment by pal1000** [comment]
It looks you didn't read the release notes before downloading or you didn't understand. If you would have done it you wouldn't have been so surprised, I can't do more about it, but I am sure Windows uses the updated driver because the driver file gets locked and Realtek Audio Console reports updated driver. Duplicate of #10.
